pyfarm/agent/etc/agent.yml

```diff
@@ -209,3 +209,5 @@ sysinfo_command_lspci:
   - /sbin/lspci
   - /usr/sbin/lspci
   - /usr/bin/lspci
+
+agent_allow_sharing: False
```

Comment:
Can we get a doc string above this?  Copying part of what's in the description of this PR would probably be fine.